### PR TITLE
Update python-base submodule

### PR DESCRIPTION
**Pull request by roycaihw**
ref https://github.com/kubernetes-client/python/issues/445

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v6.0.0b1
+- Update python-base submodule: config loader now raises exception on duplicated name in kubeconfig [kubernetes-client/python-base#47](https://github.com/kubernetes-client/python-base/pull/47)
```
Comment (mbohlool) [inline CHANGELOG.md]: I don't think we want "Update python-base submodule:". That is an implementation detail and users don't care about it.

Comment (roycaihw) [reply CHANGELOG.md]: Updated

**Comment by roycaihw** [comment]
@mbohlool Done. Thanks for reviewing this PR!

**Pull request merged**